webapi/src/main/scala/org/knora/webapi/routing/AroundDirectives.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2015-2018 the contributors (see Contributors.md).
+ *
+ *  This file is part of Knora.
+ *
+ *  Knora is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Knora is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public
+ *  License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.routing
+
+import akka.event.LoggingAdapter
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives._
+
+object AroundDirectives {
```

Comment:
Could you add a Scaladoc comment to this class?

Reply:
Sure. Will do.

Reply:
Done